angular.js

```diff
@@ -4,5 +4,33 @@ module.exports = {
         'plugin:@angular-eslint/recommended--extra',
         'plugin:@angular-eslint/template/process-inline-templates',
```

Comment:
Just to confirm, should this be used in the `angular-template.js` ruleset instead?

Reply:
No, this enables inline template linting which will use `angular-template.js` automatically.